### DISCUSSION TITLE
Enable GitHub Sponsors To Non-GitHub Users To Sponsor Us

### DISCUSSION THREAD

**Discussion by jfullstackdev** [community/community#27603] [Other Feature Feedback, Questions, & Ideas]
I tested the GitHub Sponsor by clicking the button but once I'm at the Sponsor's Page and selected a tier, I'm being asked to log-in to GitHub

unfortunately, this will discourage non-GitHub sponsors because it's only for GitHub users, the simpler the sponsoring, the more of us will be sponsored even outside GitHub,

compared to, say, PayPal, there is a certain way even if a certain donor wants to donate without PayPal, all handled by PayPal itself, hence, the ease of donating and more will be encouraged to do so

please upvote because all of us will benefit from this, imagine just sharing your donation button at Twitter and other non-GitHub user wants to be your sponsor, he / she will not be discouraged to do so

there is an effect on GitHub too, it will be competing with other services because more and more will be encouraged to create their GitHub accounts knowing this feature, much like GitHub Pages that encourages users to sign up just for this feature

**Comment by perdrix52** [comment]
I 100% agree - I was shocked the other day to find that my users couldn't sponsor my project unless they were github users.

This should be open to all.
